tests/qchem/test_tapering.py

```diff
@@ -656,11 +717,11 @@ def test_taper_excitations(
     ("operation", "op_gen", "message_match"),
     [
         (qml.U2(1, 1, 2), None, "is not implemented, please provide it with 'op_gen' args"),
-        (
-            qml.U2(1, 1, 2),
-            np.identity(16),
-            "Generator for the operation needs to be a qml.Hamiltonian",
-        ),
+        # (
+        #     qml.U2(1, 1, 2),
+        #     np.identity(16),
+        #     "Generator for the operation needs to be a qml.Hamiltonian",
+        # ),
```

Comment:
Should this be removed? Do we support  both `Hamiltonian` and `Operator`?

Reply:
Yes we support both now